recipe/build.sh

```diff
@@ -14,9 +12,6 @@ then
     export UUID_CFLAGS=" "
     export UUID_LIBS=" "
     export PKGCONFIG_REQUIRES_PRIVATELY="${PKGCONFIG_REQUIRES_PRIVATELY} uuid"
```

Comment:
Although I recently OK'ed you adding it, I am pretty sure this `PKGCONFIG_REQUIRES_PRIVATELY` line is actually not what we want. It means that `uuid` shows up in fontconfig's `pkg-config` requirements on macOS. Although it is a "private" requirement that turns out to still mean that the file `uuid.pc` must be available to `pkg-config`, which it is not on a stock system (as far as I can tell). So to build against fontconfig we end up having to install our rebuilt `libuuid` library — and the whole point is to avoid doing that. [I ended up doing a gross hack to work around this to get xorg-libxft across the finish line](https://github.com/conda-forge/xorg-libxft-feedstock/commit/8f11f2e0bc218439395c2fa631cb37a343996c8c). (Probably other approaches would have worked, but I couldn't think of anything non-gross.)

What caused you to add the line? From a pkg-config perspective, I think we want to drop it, but it could certainly be that there are conflicting requirements here.

Reply:
This was what they did in the accepted patch upstream as explained in [this comment]( https://github.com/conda-forge/fontconfig-feedstock/pull/25#discussion_r202063742 ). So this will be the future behavior of `fontconfig`. We could opt to use the patch instead, which deals with the private requirement concern you have raised.